doc/Tribler API.md

```diff
@@ -0,0 +1,31 @@
+# Tribler API
```

Comment:
Let's call it REST API, it's what it really is and it will avoid confusion with the core API.

I think the API package should be renamed to something else too.